common/djangoapps/student/models.py

```diff
@@ -294,6 +294,9 @@ class Meta(object):
     allow_certificate = models.BooleanField(default=1)
     bio = models.CharField(blank=True, null=True, max_length=3000, db_index=False)
     profile_image_uploaded_at = models.DateTimeField(null=True, blank=True)
+    mobytize_id = models.CharField(max_length=255, blank=True)
```

Comment:
I believe we need another migration for these new fields (mobytize_id, mobytize_token)

Reply:
this one: https://github.com/raccoongang/edx-platform/blob/d4f397584f679c111c0af141f7ee602a4a418597/common/djangoapps/student/migrations/0011_auto_20190726_0445.py

Reply:
common/djangoapps/student/migrations/0011_auto_20190726_1017.py added to git